conn_test.go

```diff
@@ -66,7 +66,7 @@ func createListener6(t *testing.T) *net.UDPConn {
 }
 
 func TestValidCommunication(t *testing.T) {
-	lim := test.TimeOut(time.Second * 10)
+	lim := test.TimeOut(time.Second * 30)
```

Comment:
This was very odd.

On a Mac on my local network (which has a lot of devices on) this finishes in about 25 seconds.

Reply:
In fact the entire way these tests interact with local network devices is a bit of a problem.

Reply:
This timeout also causes failure on current master.